tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_manager.h

```diff
@@ -73,6 +73,25 @@ class ProcessManager
    * @return True if successful
    */
   virtual bool clear() = 0;
+
+  /**
+   * @brief Enable/Disable debug for the process manager
+   *
+   * This will add an observer which will print out information on the task as it is executed.
+   *
+   * @param enabled
+   */
+  virtual void setDebug(bool enabled) = 0;
+
+  /**
+   * @brief Enable/Disable profiling for the process manager
+   *
+   * This will add an observer which will profile task as it is executed. It will save a file in the temp directory
+   * which can be loaded into the taskflow profile webpage.
+   *
+   * @param enabled
+   */
+  virtual void setProfile(bool enabled) = 0;
```

Comment:
Maybe enableDebug and enableProfiling? SetProfile sounds like the motion planning profiles

Reply:
I like it.